ui/src/app/what-if/what-if.component.css

```diff
@@ -0,0 +1,92 @@
+#workload_title{
```

Comment:
rename this file to .scss

Reply:
As suggested try to avoid styling as much as possible. Use the clarity grid layout and clarity components

Reply:
Ok